[electron] Add 19, Update 18.3.0, 16.2.8, 15.5.7, EoL 16, 15

BiNZGi opened the pull request:
https://github.com/electron/electron/releases/tag/v19.0.0
https://github.com/electron/electron/releases/tag/v18.3.0
https://github.com/electron/electron/releases/tag/v16.2.8
https://github.com/electron/electron/releases/tag/v15.5.7
With the release of 19 in May 2022 only 19, 18, 17 are supported see https://www.electronjs.org/blog/8-week-cadence#-will-electron-extend-the-number-of-supported-versions

BiNZGi committed:
[electron] Add 19, Update 18.3.0, 15.5.6, EoL 16, 15

https://github.com/electron/electron/releases/tag/v19.0.0
https://github.com/electron/electron/releases/tag/v18.3.0
https://github.com/electron/electron/releases/tag/v15.5.6
With the release of 19 in May 2022 only 19, 18, 17 are supported see https://www.electronjs.org/blog/8-week-cadence#-will-electron-extend-the-number-of-supported-versions

BiNZGi committed:
[electron] Update 19 release date

captn3m0 commented on products/electron.md:
```diff
@@ -17,22 +17,26 @@ sortReleasesBy: releaseCycle
     latest: "17.4.5"
   - releaseCycle: "16"
-    eol: false
+    eol: true
```
Better to put a date.

BiNZGi replied:
Thanks for the hint, updated with 8dfe4f7f4573a7a59e2412b33205af6eeee77971

BiNZGi committed:
[electron] Update eol date for 15, 16

BiNZGi committed:
[electron] Update 16.2.8, 15.5.7

https://github.com/electron/electron/releases/tag/v16.2.8
https://github.com/electron/electron/releases/tag/v15.5.7

captn3m0 committed:
Merge branch 'master' into electron

captn3m0 committed:
[electron] Fix indent
